pages/docs/index.tsx

```diff
@@ -100,31 +151,30 @@ const Docs = ({ bitbucket_auth_url }: { bitbucket_auth_url: string }) => {
 	const [progress, setProgress] = React.useState(43);
 	const [isButtonDisabled, setIsButtonDisabled] = useState<boolean>(false);
   	const [buildStatus, setBuildStatus] = useState<CloudBuildStatus | null>(null);
+	const handleBuildButtonClick = async () => {
+		setIsButtonDisabled(true);
+		setProgress(10); // Reset progress to 0 when the button is clicked
```

Comment:
We are not using progress anymore. This does not seem to be compatible with the existing code.